core/templates/dev/head/pages/library/search_results_directive.html

```diff
@@ -24,13 +24,10 @@
         <img ng-src="<[noExplorationsImgUrl]>" alt="No explorations found">
         <p style="text-align: center;">
           <span translate="I18N_LIBRARY_NO_EXPLORATION_FOR_QUERY"></span><br><br>
-          {% if not has_fully_registered %}
-            <a href ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
-               translate="I18N_LIBRARY_CREATE_EXPLORATION_QUESTION"></a>
-          {% else %}
-            <a href="/creator_dashboard?mode=create"
-               translate="I18N_LIBRARY_CREATE_EXPLORATION_QUESTION"></a>
-          {% endif %}
+          <a href ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
```

Comment:
Could you put the ng-if immediately after the leading "a", in both these directives? This will make the if/else structure more clear, I think.

Also, I think you've got the if/else conditions the wrong way round.